templates/store/snap-distro-install.html

```diff
@@ -68,7 +68,7 @@
       <div class="row">
         <div class="mobile-col-2 col-8"></div>
         <div class="mobile-col-2 col-4">
-          <img class="distro-banner__logo" src="{{ distro_logo }}" />
+          <img class="distro-banner__logo" src="{{ distro_logo_mono }}" />
```

Comment:
Now that this logo is not in 'background' anymore we could possibly move it out of here completely and have it next to heading in HTML. This may help with position it in more standard way in grid.

So I would move this whole `<div class="mobile-col-2 col-4">` with logo next to `<h1>` element. `<h1>` would need to be wrapped in `col-8` as well. So something like:

```html
<div class="row">
  <div class="col-8">
     <h1 class="p-heading--two u-no-margin is-light">How to install {{ snap_title }}<br/> on <b>{{ distro_name }}</b></h1>
  </div>
  <div class="col-4 mobile-col-2">
    <img class="distro-banner__logo" src="{{ distro_logo_mono }}" />
 </div>
</div>
```

Reply:
And we could use `u-vertically-center` then to make sure heading and logo are centrally aligned `<div class="row u-vertically-center">`